src/StratisMasternodeDashboard/Views/Home/Dashboard.cshtml

```diff
@@ -339,32 +339,29 @@
                                     <ul class="list-inline">
                                         <li class="list-inline-item"><strong>Total Peers / In / Out:</strong></li>
                                         <label id="lblSidechainTotalConnectedNode"> </label>
-                                        <li class="list-inline-item">@Model.SidechainNode.Peers.Where(p => p.Type == "inbound").Count()  /  @Model.SidechainNode.Peers.Where(p => p.Type == "outbound").Count() </li>
+                                        <label id="lblSidechainTotalInNode"> </label>
+                                        <label id="lblSidechainTotalOutNode"> </label>
                                     </ul>
                                 </li>
                             </ul>
                             <label class="mt-3"><strong>Peers :</strong></label>
-                            <table class="table table-sm table-striped" id="sidechain-peers">
+
+                            <div class="row col-sm-12" id="sidechain-peerconnections">
+                                <table class="table table-sm table-striped" id="network-connection">
                                 <thead>
                                     <tr>
                                         <th class="text-left">Endpoint</th>
-                                        <th class="text-center">Type</th>
+                                        <th class="text-center">Inbound</th>
                                         <th class="text-center">Height</th>
                                         <th style="width: 250px;" nowrap class="text-left">Version</th>
                                     </tr>
                                 </thead>
-                                <tbody>
-                                    @foreach (var peer in Model.SidechainNode.Peers)
-                                    {
-                                        <tr>
-                                            <td class="text-left">@peer.Endpoint</td>
-                                            <td class="text-center">@peer.Type</td>
```

Comment:
This should  be added or changed back as per previous comment :)

Reply:
Done.